<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoapMessageDialog</class>
 <widget class="QDialog" name="SoapMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1539</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>630</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 5px;
border-style: solid;
background-color: rgb(120, 120, 0);
color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 127);
border-radius: 25px;</string>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2a">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>700</y>
     <width>1051</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 5px;
border-style: solid;
background-color: rgb(120, 120, 0);
color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 127);

border-radius: 25px;</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>630</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 5px;
border-style: solid;
background-color: rgb(120, 120, 0);
color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 127);
border-radius: 25px;</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
image: url(:/images/back.png);
border-width: 5px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
border-radius: 25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 5px;
border-style: solid;
background-color: rgb(120, 120, 0);
color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 127);
border-radius: 25px;</string>
   </property>
   <property name="text">
    <string>REQUEST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 5px;
border-style: solid;
background-color: rgb(120, 120, 0);
color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgb(0, 0, 127);
border-radius: 25px;</string>
   </property>
   <property name="text">
    <string>RESPONSE</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>491</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 25px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>80</y>
     <width>491</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 25px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>540</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>IP Adress:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
